pkg/virt-api/rest/subresource.go

```diff
@@ -1299,6 +1299,11 @@ func (app *SubresourceAPIApp) VMRemoveVolumeRequestHandler(request *restful.Requ
 	app.removeVolumeRequestHandler(request, response, false)
 }
 
+// VMAddInterfaceRequestHandler handles the subresource for hot plugging a network interface.
```

Comment:
Can you please extract this specific command handling to a separate file with a proper name?
Same for the tests.

The existing files are too large and we know large source code files are hard to maintain.